packages/connect-extension-protocol/package.json

```diff
@@ -5,6 +5,7 @@
   "main": "dist/index.js",
   "type": "module",
   "scripts": {
+    "deep-clean": "yarn clean && rm -rf node_modules",
```

Comment:
Would it not be better to have the `yarn clean` happen as a `predeep-clean` script?

Reply:
Could be - at the end of the day its the same thing though